XenAdmin/Wizards/NewVMWizard/Page_CpuMem.cs

```diff
@@ -94,24 +94,21 @@ protected override void PageLoadedCore(PageLoadedDirection direction)
             {
                 spinnerDynMin.Initialize(Template.memory_static_max, Template.memory_static_max);
                 labelDynMin.Text = Messages.MEMORY_COLON;
-                labelDynMaxInfo.Visible = labelStatMaxInfo.Visible = labelDynMax.Visible = labelStatMax.Visible = spinnerDynMax.Visible = spinnerStatMax.Visible = false;
             }
             else
             {
                 labelDynMin.Text = Messages.DYNAMIC_MIN_COLON;
-                labelDynMaxInfo.Visible = labelDynMax.Visible = spinnerDynMax.Visible = true;
                 spinnerDynMin.Initialize(Template.memory_dynamic_min, Template.memory_static_max);
                 FreeSpinnerLimits();  // same as CA-33831
                 spinnerDynMax.Initialize(Template.memory_dynamic_max, Template.memory_static_max);
                 if (memoryMode == 3)
                 {
                     FreeSpinnerLimits();
                     spinnerStatMax.Initialize(Template.memory_static_max, Template.memory_static_max);
-                    labelStatMaxInfo.Visible = labelStatMax.Visible = spinnerStatMax.Visible = true;
                 }
-                else
-                    labelStatMaxInfo.Visible = labelStatMax.Visible = spinnerStatMax.Visible = false;
             }
+            labelDynMaxInfo.Visible = labelDynMax.Visible = spinnerDynMax.Visible = 2 <= memoryMode;

```

Comment:
Since we have replace the hardcoded numbers with enum members, it's better to use them.